### PR TITLE
option to specify minimum number of models

### PR DESCRIPTION
**Pull request by sbfnk**
At the moment we're sometimes creating ensembles consisting of just 1 model, which is not great. This adds an option to specify the number of models.

### CODE REVIEW

File: R/run_ensemble.R [hunk 1]
```diff
@@ -82,6 +86,15 @@ run_ensemble <- function(method = "mean",
     mutate(quantile = round(quantile, 3),
            horizon = as.numeric(horizon))
 
+  if (min_nmodels > 0) {
+    forecasts <- forecasts %>%
+      group_by(forecast_date, location, horizon, temporal_resolution,
+               target_variable) %>%
+      mutate(n = n_distinct(model)) %>%
+      filter(n >= min_nmodels) %>%
+      select(-n)
```
Comment (Bisaloo) [inline R/run_ensemble.R:95]:
```suggestion
      filter(n_distinct(model) >= min_nmodels)
```

**Comment by Bisaloo** [comment]
Also, tests are failing because you should run `devtools::document()`

File: R/run_ensemble.R [hunk 2]
```diff
@@ -82,6 +86,15 @@ run_ensemble <- function(method = "mean",
     mutate(quantile = round(quantile, 3),
            horizon = as.numeric(horizon))
 
+  if (min_nmodels > 0) {
```
[the remark this answers is not in the document]
Comment (sbfnk) [reply R/run_ensemble.R:89]: Good point, I'll remove it.

**Comment by sbfnk** [comment]
> Also, tests are failing because you should run `devtools::document()`

There was me thinking you'd automated it! 😉

**Pull request merged**